openreview/conference/templates/submission_revision_process.py

```diff
@@ -38,17 +38,22 @@ def process_update(client, note, invitation, existing_note):
         notes=client.get_notes(original=forum.id)
         if notes:
             bibtex_note=notes[0]
+            released_note = bibtex_note.content.get('authors') != ['Anonymous']
             bibtex_note.content = {
                 'venue': bibtex_note.content.get('venue'),
                 'venueid': bibtex_note.content.get('venueid')
             }
+            if not released_note:
+                bibtex_note.content['authors'] = ['Anonymous']
+                bibtex_note.content['authorids'] = CONFERENCE_ID + '/Paper' + str(bibtex_note.number) + '/' + AUTHORS_NAME
```

Comment:
I was hesitant to use `author_group` here because of the statement 
`if note.content.get('authorids'):`

Is there any circumstance where `authorids` might not be in the content of a note?